README.md

```diff
@@ -8,75 +8,100 @@ We more than welcome contributions to KubeVirt documentation. Please reach out i
 
 ### Get started
 
-- Create fork of GitHub user-guide repo
+1. Create fork of GitHub user-guide repo
 
-- Clone repository, check out source branch and prepare the Jekyll site
-  ```console
-  git clone -b master https://github.com/mygithubname/user-guide.git && cd user-guide
-  ```
+2. Clone repository, check out source branch and prepare the Jekyll site
 
-- Set up your git repo remotes like the following:
+```console
+$ git clone -b master https://github.com/mygithubname/user-guide.git && cd user-guide
 ```
+
+3. Set up your git repo remotes like the following:
+
+```console
 $ git remote -v
 origin	git@github.com:mygithubname/user-guide.git (fetch)
 origin	git@github.com:mygithubname/user-guide.git (push)
 upstream	git@github.com:kubevirt/user-guide.git (fetch)
 upstream	git@github.com:kubevirt/user-guide.git (push)
 ```
 
-- We use [mkdocs](https://www.mkdocs.org/) markdown engine with [mkdocs-awesome-pages](https://github.com/lukasgeiter/mkdocs-awesome-pages-plugin/) plugin
+We use [mkdocs](https://www.mkdocs.org/) markdown engine with [mkdocs-awesome-pages](https://github.com/lukasgeiter/mkdocs-awesome-pages-plugin/) plugin
   - mkdocs config file
   - Each subdirectory of `./docs` contains a `.pages` file.  We use this to force the ordering of pages.  Alphabetical ordering is not ideal for technical documentation.
 
-- Markdown lives under `./docs`.  Do your work here.
+4. Markdown lives under `./docs`.  Do your work here.
+
+5. Test your changes locally:
 
-- When finished check your work by running:
- - `make check_spelling`
- - `make check_links`
- - `make run`
+```console
+$ make check_spelling
+$ make check_links
+$ make build_image_userguide
+$ make run
+```
 
-- Web browse to http://0.0.0.0:8000 and validate page rendering
+**NOTE** If you use `docker` you may need to set `CONTAINER_ENGINE` and `BUILD_ENGINE`:
 
-- Commit your code
- - We do commit with signature verification so please do:
- `git commit -s -m "The commit message" file1 file 2 ...`
+```console
+$ CONTAINER_ENGINE=docker BUILD_ENGINE=docker make run
```

Comment:
I'm not sure I understand -- `make run` should take both of those variables from the environment? Is there some quirk to `make`'s variable processing? There's also the option to have those variables be make variables (i.e. `make run CONTAINER_ENGINE=...`.

- [x] Improve variable setting for docker `make run` example

I've gone with explicit `export` statements since I think that should hopefully clear up any issues?